hyperlane-interfaces/src/main.sw

```diff
@@ -12,6 +14,18 @@ abi Mailbox {
     #[storage(read, write)]
     fn dispatch(destination_domain: u32, recipient: b256, message_body: Vec<u8>) -> b256;
 
+    #[storage(read, write)]
```

Comment:
could we add some brief `///` comments for the functions in this fiile? I figure documenting the ABIs that ppl will be importing will be important